core/src/main/thrift/client.thrift

```diff
@@ -22,6 +22,11 @@ namespace cpp org.apache.accumulo.core.clientImpl.thrift
 include "security.thrift"
 include "trace.thrift"
 
+// Thrift Id, a wrapper for AbstractId over thrift
+struct Tid {
```

Comment:
What about `TAbstractId`? There are places in the code where we use "tid" for transactionId.

Reply:
or `idType`

Reply:
I wasn't sure if I should even keep this type at all, or if I should make two separate types, TTableId and TNamespaceId, which corresponding utility methods, to further disambiguate. I was also considering TTable and TNamespace types that would have an ID/Name pair, to make these useful to carry name information for log messages so that way we don't have to do a name lookup on the server side for a log message, and so we could see the name the client thought the table was at the time it did the ID lookup, in case the table gets renamed before the log message is printed.

In short, there's room to grow here, but I'm not 100% sure which is the best direction.

Reply:
I am in favor of the TTable/TNamespace idea that contains a pair of ID/Name.